test/runnable/test18076.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+src=runnable${SEP}extra-files
+dir=${RESULTS_DIR}${SEP}runnable
+output_file=${dir}/test18076.sh.out
+
+echo 'import std.stdio; void main() { writeln("Success"); }' | \
+	$DMD -m${MODEL} -run - > ${output_file} || exit 1
```

Comment:
Don't you have to check that it actually prints "Success" here?

```
[[ $(echo "Success") == "Success" ]] || exit 1
```

Reply:
You're right, fixed.